payments/utils.py

```diff
@@ -258,12 +273,38 @@ def resolve_area(order: Order):
         lease_order.product, "winter_storage_area"
     ):
         return lease_order.product.winter_storage_area
-    elif hasattr(lease_order.lease, "berth"):
+    elif hasattr(lease_order, "lease") and hasattr(lease_order.lease, "berth"):
         return order.lease.berth.pier.harbor
 
     return None
 
 
+def resolve_order_place(
+    lease,
+) -> Optional[Union[Berth, WinterStoragePlace, WinterStorageSection]]:
+    """
+    Resolve the place (Berth/Winter storage place) or section (Winter storage section)
+    to which an order is connected, if any.
+    """
+    return (
+        getattr(lease, "berth", None)
+        or getattr(lease, "place", None)
+        or getattr(lease, "section", None)
+    )
+
+
+def resolve_product_talpa_ecom_id(
+    product: Union[BerthProduct, WinterStorageProduct, AdditionalProduct]
+) -> str:
+    """
+    Resolves the corresponding Talpa eCom product id, according to the product type
+    and region where the associated area is located.
+    """
+    # TODO: When VEN-1414 and VEN-1413 are finished, the Talpa product ID has to be
+    #  resolved here using the ProductAccounting
```

Comment:
The code in https://github.com/City-of-Helsinki/berth-reservations/pull/626 takes an Order as a parameter instead of product, so the interface is different, can it be made to work?